guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -106,45 +157,242 @@ You can customize the version numbers by changing the `major` and `minor` settin
 
 To import a Content View, ensure that the {ProjectServer} where you want to import meets the following conditions:
 
-ifeval::["{build}" == "satellite"]
```

Comment:
This looks to me to still be relevant for disconnected scenarios. How else will they manage?